controller/controlplane/manager_options_test.go

```diff
@@ -754,3 +755,42 @@ func TestManagerConfigToStatusFeatureGates(t *testing.T) {
 		})
 	}
 }
+
+func TestWithAnonymousReports(t *testing.T) {
+	testCases := []struct {
+		name     string
+		enabled  bool
+		expected bool
+	}{
+		{"anonymous reports enabled", true, true},
+		{"anonymous reports disabled", false, false},
```

Comment:
Can we expand these each into it's own line, specifying the field name?

This prevents changes to field order causing unwanted consequences